inductiva/simulators/README.md

```diff
@@ -0,0 +1,235 @@
+```
+
+## SPlisHSPlasH simulator
+
+SPlisHSPlasH is a Smoothed-Particle Hydrodynamics (SPH) simulator that covers a wide-range of applications. The simulator is usually configured by a single file with the extension `.json`. This file contains all the information about the simulation, including the geometry, the physical properties of the fluids, the boundary conditions, the numerical parameters, and the output files. Sometimes the configuration can also use extra geometry files.
+
+To run a simulation you can follow exactly the same structure as above for DualSPHysics.
+
+### Example
+
+```python
+import inductiva
+
+inductiva.api_key = "YOUR_API_KEY"
+
+# Set simulation input directory
+input_dir = "splishsplash_example"
+
+# Initialize the Simulator
+simulator = inductiva.simulators.SplishSplash()
+
+# Run simulation with config files in the input directory
+task = simulator.run(input_dir=input_dir,
+                     sim_config_file="config.json")
+```
+
+## OpenFOAM simulator
+
+OpenFOAM is a Finite Volume method for CFD simulations with a wide-range of applications across several areas of engineering and science. OpenFOAM has an extensive
+range of features to solve anything from complex fluid flows involving chemical reactions, turbulence and heat transfer, to solid dynamics and electromagnetics.
+
+A single simulation via Inductiva API comprises several steps done via OpenFOAM - e.g., partitioning the domain, meshing, solvers and post-processing. Hence, to configure a simulation for OpenFOAM the user will need a set of configuration files that are spread across three folders:
+- `time`: containing individual files of data for particular fields, like initial values and boundary conditions that the user must specify. For example, for an initial condition at $t=0$ the initial conditions will be stored in the directory `0`.
+- `constant`: contains a full description of the case of the mesh, the physical properties for the application we are concerned, and the files describing some of the objects present inside the simulation and that need to be pre-processed to obtain a meshe.
+- `system`: contains all of the files that describe the simulation, including the solvers, the numerical parameters, and the output files. It must contain at least 3 files: `controlDict` where run control parameters are set including start/end time, time step and parameters for data output; `fvSchemes` where discretisation schemes used in the solution may be selected at run-time; and `fvSolution` where the equation solvers, tolerances and other algorithm controls are set for the run.
+
+All of these folders should be inside an input directory. Finally, to run a simulation the user needs to configure in list of dictionaries the commands that he wants to perform in the backend. Below, we run the [motorbike tutorial](https://github.com/OpenFOAM/OpenFOAM-8/tree/master/tutorials/incompressible/simpleFoam/motorBike) from OpenFOAM and show how this is done in practice.
```

Comment:
needs to configure a list of...

Reply:
"Finally, to run a simulation, the user needs to configure a list of dictionaries specifying the commands they want to execute on the backend."